include/ebpf_structs.h

```diff
@@ -176,17 +176,7 @@ enum bpf_prog_type
      *
      * **Helpers available:** all helpers defined in bpf_helpers.h
      */
-    BPF_PROG_TYPE_XDP,
-
-    /** @brief Program type for handling socket bind() requests.
-     *
-     * **eBPF program prototype:** \ref bind_hook_t
-     *
-     * **Attach type(s):** \ref BPF_ATTACH_TYPE_BIND
-     *
-     * **Helpers available:** all helpers defined in bpf_helpers.h
-     */
-    BPF_PROG_TYPE_BIND, // TODO(#333): replace with cross-platform program type
+    BPF_PROG_TYPE_XDP = 6,
```

Comment:
Why do we need the same literal values?  Discussed with John Fastabend in the triage meeting this morning.  Changing the value is a breaking change (in the sense that existing compiled apps might not work) and we don't think there is a reason the same binary value is needed.

Reply:
These constants are used to specify the program type in bpf() BPF_PROG_LOAD. If we don’t change them to match Linux we’d need separate definitions for Windows. It’s doable but makes the bpf() wrapper not truly ABI compatible.

There is a related problem where it’s not clear how to load windows specific program types via the shim. I need to think about that, maybe both problems can be solved in the same fashion.

Would it be possible to use separate constants for bpf() to avoid breaking code that doesn’t use the wrapper?